Gems/ScriptCanvas/Code/Include/ScriptCanvas/Grammar/ParsingUtilities.cpp

```diff
@@ -145,7 +145,7 @@ namespace ParsingUtilitiesCpp
             m_result += "\n";
         }
 
-        void EvaluateRoot(ExecutionTreeConstPtr node, const Slot*)
+        void EvaluateRoot(ExecutionTreeConstPtr, const Slot*)
```

Comment:
Just a question: is there a reason, why you removed the variable name here instead of using maybe_unused?